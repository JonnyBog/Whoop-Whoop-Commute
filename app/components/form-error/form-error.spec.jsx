import React from 'react';
import renderer from 'react-test-renderer';

import FormError from 'components/form-error/form-error';

describe('Components', () => {
  describe('FormError', () => {
    it('should export', () => {
      expect(FormError).toBeDefined();
    });

    /**
     * createRenderTree
     * @returns {JSX} component tree
     */
    function createRenderTree () {
      return renderer.create(
        <FormError />
      ).toJSON();
    }

    describe('Appearance', () => {
      it('should render', () => {
        const tree = createRenderTree();
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
