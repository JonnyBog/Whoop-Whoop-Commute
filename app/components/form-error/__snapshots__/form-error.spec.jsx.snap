// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components FormError Appearance should render 1`] = `
<p
  className="form-error__StyledFormError-mbqx9k-0 cKaLxn"
/>
`;
